packages/onegraph-apollo-server-auth/src/index.js

```diff
@@ -50,7 +50,12 @@ class hasRoleDirective extends SchemaDirectiveVisitor {
 
 class isAuthenticatedDirective extends SchemaDirectiveVisitor {
   visitFieldDefinition(field) {
-    const resolver = field.resolve;
+    const resolver =
```

Comment:
Also added this while pairing with @sgrove - prevents error if `field.resolve` is not defined.